var/spack/repos/builtin/packages/flux-pmix/package.py

```diff
@@ -57,8 +57,8 @@ def add_pluginpath(self):
 
     def setup_run_environment(self, env):
         spec = self.spec
-        env.prepend_path("FLUX_SHELL_RC_PATH", join_path(self.prefix, "etc/flux/shell/lua.d"))
+        env.prepend_path("FLUX_SHELL_RC_PATH", join_path(self.prefix.etc, "flux/shell/lua.d"))
```

Comment:
If you want to go that route, you should be able to use `join_path(self.prefix.etc.flux.shell, "lua.d")`.

Reply:
From my point of view, this change isn't desirable and not worth the cost of a CI re-run.